ui/app/src/modules/Preview/Tools/BrowseComponentsPanel.tsx

```diff
@@ -0,0 +1,328 @@
+              {
+                contentTypes.map((contentType: ContentType, i: number) => {
+                  return <MenuItem value={contentType.id} key={i}>{contentType.name}</MenuItem>
+                })
+              }
+            </Select>
+          }
+        </div>
+        <React.Suspense
+          fallback={
+            <LoadingState
+              title={formatMessage(translations.loading)}
+              graphicProps={{ width: 150 }}
+            />
+          }
+        >
+          <BrowsePanelUI
+            componentsResource={resource}
+            classes={classes}
+            onPageChanged={onPageChanged}
+            onDragStart={onDragStart}
+            onDragEnd={onDragEnd}
+            AUTHORING_BASE={AUTHORING_BASE}
+          />
+        </React.Suspense>
+      </ErrorBoundary>
+    </ToolPanel>
+  );
+
+}
+
+function BrowsePanelUI(props) {
+  const {
+    componentsResource,
+    classes,
+    onPageChanged,
+    onDragStart,
+    onDragEnd,
+    AUTHORING_BASE,
+  } = props;
+  const { formatMessage } = useIntl();
+  const components: ComponentResource = componentsResource.read();
+  const { count, pageNumber, items, limit, contentTypeFilter } = components;
+  return (
+    <div className={classes.browsePanelWrapper}>
+      {
+        contentTypeFilter ? (
+          <>
+            <TablePagination
+              className={classes.pagination}
+              classes={{ root: classes.pagination, selectRoot: 'hidden', toolbar: classes.toolbar }}
+              component="div"
+              labelRowsPerPage=""
+              count={count}
+              rowsPerPage={limit}
+              page={pageNumber}
+              backIconButtonProps={{
+                'aria-label': formatMessage(translations.previousPage),
+              }}
+              nextIconButtonProps={{
+                'aria-label': formatMessage(translations.nextPage),
+              }}
+              onChangePage={(e: React.MouseEvent<HTMLButtonElement>, page: number) => onPageChanged(e, page * limit)}
+            />
+            <List className={classes.list}>
+              {
+                items.map((item: ContentInstance) =>
+                  <PanelListItem
+                    key={item.craftercms.id}
+                    primaryText={item.craftercms.label}
+                    onDragStart={() => onDragStart(item)}
+                    onDragEnd={onDragEnd}
+                  />
+                )
+              }
+            </List>
+            {
+              count === 0 &&
+              <EmptyState
+                title={formatMessage(translations.noResults)}
+                classes={{ image: classes.noResultsImage, title: classes.noResultsTitle }}
+              />
+            }
+          </>
+        ) : (
+          <EmptyState
+            title={formatMessage(translations.chooseContentType)}
+            image={`${AUTHORING_BASE}/static-assets/images/choose_option.svg`}
```

Comment:
Switch to `chooseContentTypeImageUrl` prop. Then we won't need the AUTHORING_BASE at all.